pipelines/spancat-datasets/README.md

```diff
@@ -0,0 +1,93 @@
+<!-- SPACY PROJECT: AUTO-GENERATED DOCS START (do not remove) -->
+
+# 🪐 spaCy Project: Spancat datasets
```

Comment:
Thinking if we should rename this project since it includes both NER and Spancat. Maybe "Span labelling datasets"? Something more general?